content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -70,6 +70,39 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
 
 {{< /note >}}
 
+### Enable dual-stack via kubeadm
+
+Install kubeadm following the steps from the [Installing Kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) documentation.
+
+Make sure that nodes allow IPv6 forwarding, if not, run `sudo sysctl -w net.ipv6.conf.all.forwarding=1` on every node in the cluster.
+
+* Notice that `kubeadm upgrade` will change IPv6DualStack to true by default if feature gate is not set in old cluster. However, cluster-cidr and service-cidr modification are not supported.
+
+#### Create a dual-stack cluster
+
+You must pass your `podCidr` and `serviceCidr` value to `kubeadm init`, like so:
+
+```
+kubeadm init --feature-gates IPv6DualStack=true --pod-network-cidr=172.30.0.0/16,fefe:ffff:0::/48 --service-cidr=172.31.0.0/16,fefe:ffff:1::/108
```

Comment:
i think we should just pass `--config` with the example config we discussed as the primary example target in this guide.
these kubeadm flags are GA / stable, but we recommend config to users since it's versioned.
we can still include examples with the flags as the second option if you'd like.